### PR TITLE
Button UI component

### PR DESCRIPTION
**Pull request by guganabu**

### CODE REVIEW

File: app/components/ak-button/index.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.btn {
+  height: 2.5rem;
+  padding: 0.57em 0.85em;
```
Comment (cosmosgenius) [inline app/components/ak-button/index.scss:3]: this padding number is so weird. where is this coming from?
also height should be em not rem as it should depend on the font size of the text used not root font size

File: app/components/ak-button/index.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  font-size: 1rem;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: center;
+  column-gap: 0.5em;
+  line-height: 2.5rem;
+  cursor: pointer;
+
+  span {
```
Comment (cosmosgenius) [inline app/components/ak-button/index.scss:15]: use class there no direct reference to tag

File: app/components/ak-button/index.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  padding: 0.57em 0.85em;
+  border: none;
+  border-radius: var(--button-border-radius);
+  font-size: 1rem;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: center;
+  column-gap: 0.5em;
+  line-height: 2.5rem;
```
Comment (cosmosgenius) [inline app/components/ak-button/index.scss:12]: are these to even required?

File: app/components/ak-button/index.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  cursor: pointer;
+
+  span {
+    //Icon size
+    font-size: 1rem;
```
Comment (cosmosgenius) [inline app/components/ak-button/index.scss:17]: isn't this inherited from parent?

File: app/components/ak-button/neutral-outlined/index.hbs [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<button
+  type="button"
+  disabled={{@isDisabled}}
+  local-class="neutral-outlined"
+  ...attributes
+  data-test-btn-neutral-outlined
+  {{on 'click' @onClick}}
+>
+  {{#if @icon}}
```
Comment (cosmosgenius) [inline app/components/ak-button/neutral-outlined/index.hbs:9]: this should not exist here...it should be yielded along with text

**Review by cosmosgenius — CHANGES_REQUESTED** [review]

File: app/components/ak-button/index.scss (hunk 1 above)
Comment (guganabu) [reply app/components/ak-button/index.scss]: 1. was tried to derive it, but having 0.5em is better 👍 
2. updated height unit

File: app/components/ak-button/index.scss (hunk 2 above)
Comment (guganabu) [reply app/components/ak-button/index.scss]: yes, earlier it was a problem, now it is not anymore required!

File: app/components/ak-button/index.scss (hunk 3 above)
Comment (guganabu) [reply app/components/ak-button/index.scss]: yes, it required to be overwrite with global style which is 24px set to button element. which is ideally required to align the inner content at center by pos.

File: app/components/ak-button/index.scss (hunk 4 above)
Comment (guganabu) [reply app/components/ak-button/index.scss]: yep, wanted to not change default properties, but makes sense to update... and done!

File: app/components/ak-button/neutral-outlined/index.hbs (hunk 5 above)
Comment (guganabu) [reply app/components/ak-button/neutral-outlined/index.hbs]: It was in thought process but later on decided to wrap the contents within component itself. Moved now

**Review by cosmosgenius — APPROVED** [review]

**Pull request merged**